scripts/apps/authoring/authoring/directives/ItemAssociationDirective.js

```diff
@@ -24,8 +24,9 @@ export function ItemAssociationDirective(superdesk, renditions, config, authorin
             rel: '=',
             item: '=',
             editable: '=',
-            allowVideo: '=',
             allowPicture: '=',
+            allowVideo: '=',
```

Comment:
Since AngularJS 1.5 we can use [one-way bindings](https://docs.angularjs.org/api/ng/service/$compile#-scope-). It's not necessary to bind this two-ways. Would you mind replacing these?
```js
editable: '<',
allowVideo: '<',
allowPicture: '<',
allowAudio: '<',
showTitle: '<'
```
Thanks!